src/app/common/default.controller.js

```diff
@@ -0,0 +1,52 @@
+
+(function() {
+  'use strict';
+  angular
+    .module('healthGuide')
+    .controller('defaultController', defaultController);
+
+  function defaultController(
+                      localForageService,
+                      sectionsService,
+                      sidenavService,
+                      userService
+                      ) {
+      "ngInject";
+      /*jshint validthis: true*/
+      var self = this;
+      self.state = 'default';
+
+      self.toggleSideNav = sidenavService.toggleList;
+      self.openLeftMenu = sidenavService.openLeftMenu;
+      self.closeSidenav = sidenavService.closeSidenav;
+
+      self.user = userService.user;
+      self.data = sectionsService.data[self.state];
+      self.calculate = calculate;
+
+      activate();
+
+      function activate() {
+        self.closeSidenav();
+        if (_.isEqual(userService.user, userService.mockUser)) {
```

Comment:
The solution.

[It's the best solution ](https://github.com/VasylPosmit/health/pull/26/commits/f896199d9b109efb36deeb6824e44f3f4f2e5cfe#diff-ba3a8e9b2091a4198eaeca2956c0e0c7R32)of about 10 that I had implemented 

It's about ok.